src/plone/restapi/tests/http-examples/siteroot.resp

```diff
@@ -15,6 +15,7 @@ Content-Type: application/json
   }, 
```

Comment:
While the http examples have been updated, the narrative documentation seems to be out of date. https://plonerestapi.readthedocs.io/en/latest/tiles.html needs a major update I think. I'm honestly quite confused at this point when trying to determine which *tiles* related functionality is implemented, and where, and where the documentation is just stubbed out, talks about future ideas that may or may not happen, or is outdated.

Please, please try to keep the documentation in sync with API changes - it's so very hard to even to attempt to do it after the fact, after things have changed over time.

Reply:
@lukasgraf you are absolutely correct. Thanks for pointing that out. I created a separate issue for that since it is not directly related to the PR. https://github.com/plone/plone.restapi/issues/674